### PR TITLE
[4.0] Namespace Editors - TinyMCE plugin

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Namespaces Editors - TinyMCE plugin.

### Testing Instructions

Delete `libraries/autoload_psr4.php` file.
Edit `Editor - TinyMCE` plugin options.
Check that TinyMCE builder appears.
Check that `Site Skin` and `Administrator Skin` selectors appear correctly inside the builder.
Check that `Images Directory` selector appears correctly.

### Documentation Changes Required

IDK.

### CODE REVIEW

File: plugins/editors/tinymce/src/Field/UploaddirsField.php [hunk 1]
```diff
@@ -20,14 +22,14 @@
...
+class UploaddirsField extends FolderlistField
 {
 	protected $type = 'uploaddirs';
 
 	/**
 	 * Method to attach a JForm object to the field.
 	 *
-	 * @param   SimpleXMLElement  $element  The SimpleXMLElement object representing the `<field>` tag for the form field object.
+	 * @param   \SimpleXMLElement  $element  The SimpleXMLElement object representing the `<field>` tag for the form field object.
 	 * @param   mixed             $value    The form field value to validate.
```
Comment (Quy) [inline plugins/editors/tinymce/src/Field/UploaddirsField.php]: Align 3rd column.

**Comment by SharkyKZ** [comment]
Conflicts solved.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on b55606c1df896cb34251dc88da013ab6bd0fd9d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27648">issues.joomla.org/tracker/joomla-cms/27648</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on b55606c1df896cb34251dc88da013ab6bd0fd9d5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27648">issues.joomla.org/tracker/joomla-cms/27648</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27648">issues.joomla.org/tracker/joomla-cms/27648</a>.</sub>

**Comment by richard67** [comment]
Grr bloody issue tracker adds back the conflicting files label. Have to fix it here, but then it will still be shown in the tracker.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**